GeometryCore/src/Mesh.cpp

```diff
@@ -208,4 +210,52 @@ namespace geom_utils
 		}
 		return foundation;
 	}
+	Mesh integrateFoundationIntoModel(const Mesh& model,const Mesh& foundation)
+	{
+		Mesh resultFigure;
+		Mesh copyModel(model);
+		Mesh copyFoundation(foundation);
+		if (!model.isEmpty() && !foundation.isEmpty())
+		{
+			auto zDiff = copyModel.getAABB().pmin.z - copyFoundation.getAABB().pmax.z;
+			const float gap = 0.5;
+			zDiff -= gap;
+			copyFoundation.translate(FPoint3D(0, 0, zDiff));
+			resultFigure.merge(model);
+			resultFigure.merge(copyFoundation);
+		}
+		return resultFigure;
+	}
+	void Mesh::merge(const Mesh& figure)
```

Comment:
This is additional. If you will have time and inspiration))
Try to define `operator+` and `operator+=` for Mesh class, which will do the same as merge() does. So that we would be able to run the code:
```
Mesh one;
Mesh two;
Mesh three = one + two;
one += two;
```

Reply:
Should I delete the mesh method?

Reply:
No, just provide the additional interface for the user through the operator+, operator+=